client-spark/common/src/main/java/org/apache/spark/shuffle/reader/RssShuffleDataIterator.java

```diff
@@ -38,6 +38,7 @@
 import org.apache.uniffle.client.api.ShuffleReadClient;
 import org.apache.uniffle.client.response.CompressedShuffleBlock;
 import org.apache.uniffle.common.RssShuffleUtils;
+import sun.nio.ch.DirectBuffer;
```

Comment:
Should we use ` java.nio.DirectBuffer`?

Reply:
I think there is no need to use `DirectBuffer` here, use `HeapByteBuffer` when decompress is better, in our internal version, we can use `HeapByteBuffer` or `DirectBuffer`,  and we found no apparent difference in performance.

Reply:
Should we  add a config option to decide which ByteBuffer to use?

Reply:
No, we add this config option just for test and comparison